### ISSUE TITLE
Is there an issue in _unwind final require statement ?

### ISSUE THREAD

**Issue by kassandraoftroy** [hexonaut/guni-lev#2]
https://github.com/hexonaut/guni-lev/blob/master/src/GuniLev.sol#L393

wont `otherToken.balanceOf(address(this))` in require statement always be 0 because you just transferred it to `usr` ? As far as I understand it should just be dust so it shouldn't matter much but I think you want to do that balance reading before the transfer.

**Comment by hexonaut** [comment]
Yeah it should be the address of the user, but as you say it's either zero or dust.
